ibm/resource_ibm_private_dns_glb_monitor.go

```diff
@@ -57,7 +57,7 @@ func resourceIBMPrivateDNSGLBMonitor() *schema.Resource {
 				Type:        schema.TypeString,
 				Required:    true,
 				ForceNew:    true,
-				Description: "The GUID of the private DNS. ",
```

Comment:
Keep it has the `The GUID of the private DNS.`

Reply:
done